core/server/data/migrations/versions/3.24/01-populate-group-for-new-portal-settings.js

```diff
@@ -0,0 +1,39 @@
+const logging = require('../../../../../shared/logging');
+
+// new setting keys and group mapping
+const groupMapping = [{
```

Comment:
I don't think there's a need to have this as a one item array? NABD tho

Reply:
Same as [here](https://github.com/TryGhost/Ghost/pull/12034#discussion_r452760978), the primary reason for this structure of test is allowing easier reuse in future for n number of new settings with different groups, that only need to update this mapping.

Reply:
cool okay